napalm_ros/ros.py

```diff
@@ -305,45 +305,32 @@ def get_environment(self):
         }
 
         try:
-            system_health = tuple(self.api('/system/health/print'))[0]
-        except IndexError:
-            return environment
-
-        if system_health.get('active-fan', 'none') != 'none':
-            environment['fans'][system_health['active-fan']] = {
-                'status': int(system_health.get('fan-speed', '0RPM').replace('RPM', '')) != 0,
-            }
-
-        if 'temperature' in system_health:
-            environment['temperature']['board'] = {
-                'temperature': float(system_health['temperature']),
-                'is_alert': False,
-                'is_critical': False,
-            }
-
-        if 'cpu-temperature' in system_health:
-            environment['temperature']['cpu'] = {
-                'temperature': float(system_health['cpu-temperature']),
-                'is_alert': False,
-                'is_critical': False,
-            }
-
-        for cpu_values in self.api('/system/resource/cpu/print'):
-            environment['cpu'][cpu_values['cpu']] = {
-                '%usage': float(cpu_values['load']),
-            }
-
-        try:
+            system_health = self.api('/system/health/print')
             system_resource = tuple(self.api('/system/resource/print'))[0]
         except IndexError:
-            return {}
+            return environment
 
         total_memory = system_resource.get('total-memory')
         free_memory = system_resource.get('free-memory')
-        environment['memory'] = {
-            'available_ram': total_memory,
-            'used_ram': int(total_memory - free_memory),
-        }
+        environment['memory'] = {'available_ram': total_memory, 'used_ram': int(total_memory - free_memory)}
+
+        for entry in system_health:
```

Comment:
```suggestion
        for entry in self.api('/system/health/print'):
```
No need for line 308.